src/components/Counter.js

```diff
@@ -20,20 +22,20 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment()/* Fill me in */ }>
```

Comment:
Another way to do this is by passing the function directly `<button onClick={this.props.increment} >` because the anonymous function isn't necessary here 😊